metadata/private/providers/package_attribute_info.bzl

```diff
@@ -0,0 +1,26 @@
+"""Declares provider `PackageAttributeInfo`."""
+
+PackageAttributeInfo = provider(
+    doc = """
+Provider for declaring metadata about a Bazel package.
+""".strip(),
+    fields = {
+        "files": """
+A [depset](https://bazel.build/rules/lib/builtins/depset) of
+[File](https://bazel.build/rules/lib/builtins/File)s containing information
+about this attribute.
+""".strip(),
```

Comment:
Is the intention to have `files` contain things like data files that may relate to a particular package attribute? i.e `LICENSE.txt` files that the license metadata would reference in its `metadata.json` file?

If so, I like this over the current rules_license implementation -- I've hit issues where packages are dual-licensed or have no LICENSE.txt file. Supporting those usecases with the current rules_license implementation is a bit odd.

Reply:
Yes